src/azure-cli/azure/cli/command_modules/synapse/manual/operations/sqlpoolsensitivitylabel.py

```diff
@@ -69,8 +69,8 @@ def sqlpool_sensitivity_label_update(
         sensitivity_label.information_type = current_label.information_type
         sensitivity_label.information_type_id = current_label.information_type_id
 
-    except CloudError as ex:
-        if not (ex.error and ex.error.error and 'SensitivityLabelsLabelNotFound' in ex.error.error):
+    except HttpResponseError as ex:
+        if not (ex.status_code and ex.status_code == '404'):
```

Comment:
if not (ex and 'SensitivityLabelsLabelNotFound' in str(ex)):

Reply:
updated as request, pls review again